lib/bundler.js

```diff
@@ -76,10 +97,10 @@ function buildBundlesForFile(filename, bundleConfigs, options) {
 
 			if(bundleConfigsForFile.length) {
 				bundleConfigsToBuild = bundleConfigsForFile;
-				options.logger.info(`Found ${filename} in ${bundleConfigsForFile.length} bundle(s).`);
+				options.logger.info(`Found "${filename}"" in ${bundleConfigsForFile.length} bundle(s).`);
```

Comment:
Is the extra quotation mark after `${filename}` intentional?

Reply:
Nope! Thanks for catching that.